src/libs/odrl/refinements.js

```diff
@@ -0,0 +1,35 @@
+export const operandList = [
```

Comment:
We should think about how to actually load relevant data dynamically for the given action.